### PR TITLE
[cluster-test] enable gas price into cluster test, which allow to setup non-zero gas price

### PR DESCRIPTION
**Pull request by snakech**
## Motivation

enable configurable non zero gap price in cluster test
default gas_price = 1, use ```-E GAS_UNIT_PRICE=1``` to set non-zero gas price

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

## Test Plan

tested with gas price = 1 with both swarm and cti:
```
cargo run -p cluster-test -- --mint-file "/var/folders/qd/tg50mr0d7k75qvztr_8yk2km0000gn/T/c81ae98e57fdc3fd55f563d37c7eb3fa/mint.key" --swarm --peers "localhost:49536" --emit-tx --workers-per-ac 1
```
```
./scripts/cti -E GAS_UNIT_PRICE=1 --pr 5414 --run bench
```

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/libra/website, and link to your PR here.)

### CODE REVIEW

File: testsuite/cluster-test/src/tx_emitter.rs [hunk 1]
```diff
@@ -107,6 +108,14 @@ pub struct EmitJobRequest {
     pub thread_params: EmitThreadParams,
 }
 
+pub static GAS_UNIT_PRICE: Lazy<u64> = Lazy::new(|| {
+    if let Ok(v) = env::var("GAS_UNIT_PRICE") {
+        v.parse().expect("Failed to parse GAS_UNIT_PRICE")
+    } else {
+        0_u64
```
Comment (andll) [inline testsuite/cluster-test/src/tx_emitter.rs:115]: Can we set default to `1`?

Comment (snakech) [reply testsuite/cluster-test/src/tx_emitter.rs:115]: observed 20% to 30% perf regression if we default gas_price = 1, just set it back to 0 and we can change this in future. Good news is I confirmed with platform team which this perf regression is expected

**Pull request closed without merging**